tests/tocheck-database/Mvc/Model/Resultset/GetFirstCest.php

```diff
@@ -41,7 +43,11 @@ public function mvcModelResultsetGetFirst(IntegrationTester $I)
         /**
          * Setup the table
          */
-        (new ObjectsMigration())($this->container->get('db'));
+        $oMigration = new ObjectsMigration($this->container->get('db'));
```

Comment:
Can you use full name like $objectMigration and $stuffMigration looks like an Hungarian notation now

Reply:
@ruudboon we can leave this here, these tests needs to be rechecked after the refactoring with the database suite so we can correct it when we start moving stuff around.

Reply:
+1